vkcs/images_image.go

```diff
@@ -190,6 +190,13 @@ func resourceImagesImageUpdateComputedAttributes(_ context.Context, diff *schema
 						newProperties[oldKey] = v
 					}
 				}
+
+				// direct_url is provided by some storage drivers.
```

Comment:
which ones provide it?

Reply:
This is part of code from openstack provider, we have direct_url be set in properties by cloud and need to correctly handle it